examples/express-mysql/index.js

```diff
@@ -0,0 +1,157 @@
+/*
+Copyright 2023 SolarWinds Worldwide, LLC.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+const process = require("node:process")
+const express = require("express")
+const mysql = require("mysql2")
+const winston = require("winston")
+
+const logger = winston.createLogger({
+  transports: [new winston.transports.Console()],
+})
+
+const conn = mysql.createConnection({
+  host: "mysql",
+  user: "root",
+  password: "root",
+  database: "todo",
```

Comment:
I see now this is what it's named and is completely fine but the name did confuse me for a moment! 😄